airbyte-integrations/connectors/source-e2e-test/src/main/java/io/airbyte/integrations/source/e2e_test/DummySource.java

```diff
@@ -0,0 +1,96 @@
+package io.airbyte.integrations.source.e2e_test;
+
+import com.fasterxml.jackson.core.util.MinimalPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.SequenceWriter;
+import io.airbyte.cdk.integrations.BaseConnector;
+import io.airbyte.cdk.integrations.base.*;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.util.AutoCloseableIterator;
+import io.airbyte.protocol.models.v0.*;
+
+import java.util.List;
+
+
+public class DummySource extends BaseConnector implements Source {
+
+    public static void main(final String[] args) throws Exception {
+        final CliParse integrationCliParser = new CliParse();
+        final IntegrationConfig parse = integrationCliParser.parse(args);
+
+        final DummySource source = new DummySource();
+
+        switch (parse.getCommand()) {
+            case SPEC, DISCOVER, CHECK -> {
+                new IntegrationRunner(source).run(args);
+            }
+            case READ -> {
+                try (AutoCloseableIterator<AirbyteMessage> iterator = source.read()) {
+                    try (SequenceWriter seqWriter = DummyIterator.OBJECT_MAPPER
+                            .writerFor(AirbyteMessage.class)
+                            .with(new MinimalPrettyPrinter(System.lineSeparator()))
+                            .writeValues(System.out)) {
```

Comment:
Using a writer and writing directly to the System.out output stream. This avoids 
1. Conversion of message to string since we dont have to use System.out.println
2. Allows us to tell JACKSON that we are only going to write JSON that represents AirbyteMessage class which allows it to optimise things